OpenRA.Mods.Common/UtilityCommands/ExtractLanguageStringsCommand.cs

```diff
@@ -38,7 +38,7 @@ void IUtilityCommand.Run(Utility utility, string[] args)
 			foreach (var filename in modData.Manifest.ChromeLayout)
 			{
 				modData.ModFiles.TryGetPackageContaining(filename, out var package, out var name);
-				name = package.Name + "/" + name;
+				name = Platform.UnresolvePath(package.Name) + "/" + name;
```

Comment:
`MiniYaml.FromFile` doesn't understand how to resolve unresolved paths, so this will likely introduce edge cases that will crash the command. Unresolve the label in the comment, not the path!

Reply:
I forgot that this is used below. Fixed.